opentelemetry-sdk/src/metrics/internal/exponential_histogram.rs

```diff
@@ -100,9 +100,15 @@ impl<T: Number> ExpoHistogramDataPoint<T> {
             if (self.scale - scale_delta as i8) < EXPO_MIN_SCALE {
                 // With a scale of -10 there is only two buckets for the whole range of f64 values.
                 // This can only happen if there is a max size of 1.
-                opentelemetry::global::handle_error(MetricsError::Other(
-                    "exponential histogram scale underflow".into(),
-                ));
+                otel_error!(
```

Comment:
I don't know the inner workings enough to give a strong opinion - but unless this is a auto recoverable error, this can flood the error log.

Reply:
As per what I can understand this part of code, this error occurs with restrictive `max_size` configuration, while the application is recording measurements with values that are far apart than what allowed by `max_size`.  And error would be logged whenever the faulty measurement is recorded. If these faulty measurements are not frequent, the error log won't be flooded, else it can. Again, either some kind of throttling or simply flag to log only once need to be added. Let me know what you suggest, else I can keep TODO to revisit.

Reply:
Unless we are 100% sure this cannot cause flooding of logs, lets remove the log from here, and leave a TODO to add logging once we understand more.